charts/incubator/soft-serve/values.yaml

```diff
@@ -0,0 +1,43 @@
+image:
+  repository: tccr.io/truecharts/soft-serve
+  pullPolicy: IfNotPresent
+  tag: 0.4.4@sha256:758d04854964a10967c0895e3abca055ad569c8630664e3efe5907003c1647fc
+
+podSecurityContext:
+  runAsGroup: 0
+  runAsUser: 0
+
+securityContext:
+  readOnlyRootFilesystem: false
+  runAsNonRoot: false
+
+secretEnv:
+  SOFT_SERVE_INITIAL_ADMIN_KEY: "test123"
+
+env:
+  SOFT_SERVE_BIND_ADDRESS: 0.0.0.0
+  SOFT_SERVE_PORT: "{{ .Values.service.main.ports.main.port}}"
+  SOFT_SERVE_HOST: "http://localhost"
+  SOFT_SERVE_KEY_PATH: /.ssh
+  SOFT_SERVE_REPO_PATH: /.repos
```

Comment:
It's gonna need configmap/secret for https://github.com/charmbracelet/soft-serve#configuration
So it might be a good idea to move those over.

Reply:
Looks like this configuration file is generated at startup by the application and is configured/managed via a git repo.
So it's not going to be managed from the app

Reply:
wait really? so we cant configure it in a configmap?

Reply:
ENVS are still applying.

The host/port in the config file/repo is only for the external access (not the container's bindings).

Also the configfile is updated each time someone creates a repo (at least thats what I understand).
So you can't have it readonly